src/Chromatogram.js

```diff
@@ -218,6 +221,18 @@ export class Chromatogram {
     return applyLockMass(this, mfs, options);
   }
 
+  getMatrix(range = {}) {
+    return getSubMatrix(this, range);
+  }
+
+  estimateNbPureComponents(range = {}) {
+    return estimateNbPureComponents(this, range);
+  }
```

Comment:
Are those methods individually useful to the end user?

Reply:
If so, they should be tested on their own

Reply:
For me the getMatrix method yes, I am not pretty sure about the estimation of number of pure components. @lpatiny could have a better idea about it.